setup.py

```diff
@@ -40,6 +41,11 @@
       author_email='john.parkhill@gmail.com',
       license='GPL3',
       packages=['TensorMol'],
+      install_requires=[
```

Comment:
We should update the url, email too

Reply:
Ah! I've finally found a use for the `maintainer` field that distutils allows for. Updated with that info.